packages/invoices/src/InboundShipment/DetailView/columns.ts

```diff
@@ -121,9 +123,11 @@ export const useInboundShipmentColumns = (): Column<
           accessor: ({ rowData }) => {
             if ('lines' in rowData) {
               const { lines } = rowData;
-              return ifTheSameElseDefault(lines, 'sellPricePerPack', '');
+              return c(
+                ifTheSameElseDefault(lines, 'sellPricePerPack', '')
+              ).format();
             } else {
-              return rowData.sellPricePerPack;
+              return c(rowData.sellPricePerPack).format();
```

Comment:
Yeah this has been on my mind! `useCurrencyFormat` is a hook used like

```
const currencyString = useCurrencyFormat(line.sellPricePerPack)
```

So it can't be used here

having `c` format straight away would be nice, like the other hooks so I can do that, but I also wanted a way to 'create' currency objects 🤔